acceptance/suites/puppet5_tests/puppet5_version_test.rb

```diff
@@ -0,0 +1,20 @@
+test_name "Testing Master/Agent backwards compatibility"
+
+# Agent running on the master is current, not legacy.
+legacy_agents = agents.reject { |agent| agent == master }
+
+step "Check that legacy agents have Puppet 5.x installed"
+on(legacy_agents, puppet("--version")) do
+  assert_match(/\A5\./, stdout, "puppet --version does not start with major version 5.")
+end
+
+step "Check that Puppet Server has Puppet 6.x installed"
+on(master, puppet("--version")) do
+  assert_match(/\A6/, stdout, "puppet --version does not start with major version 6.x or 6.x")
```

Comment:
"6.x or 6.x"

Reply:
also should we tighten the check this is copied from to only be 6?

Reply:
That seems right, yeah.